src/sentry/api/endpoints/organization_dashboard_widget_details.py

```diff
@@ -1,76 +1,24 @@
 from __future__ import absolute_import
 
-from django.db import transaction
 from rest_framework.response import Response
 
-from sentry.api.bases.dashboard import OrganizationDashboardWidgetEndpoint
-from sentry.api.serializers import serialize
+from sentry.api.bases import OrganizationEndpoint
 from sentry.api.serializers.rest_framework import DashboardWidgetSerializer
-from sentry.models import DashboardWidgetQuery
 
 
-class OrganizationDashboardWidgetDetailsEndpoint(OrganizationDashboardWidgetEndpoint):
-    def delete(self, request, organization, dashboard, widget):
```

Comment:
You're removing delete?

Reply:
Yes, widgets are removed when the dashboard is updated with DashboardDetailsEndpoint.put. I missed this endpoint when doing cleanup last week.